csds-client/client/v3/client.go

```diff
@@ -0,0 +1,390 @@
+// Package client/v3 implements the client interface for v3 transport api version in specific
```

Comment:
"in specific" - incomplete sentence? probably can be deleted. Just "Package v3/client implements the client interface for v3 transport api version"